engine/src/exec-interface.cpp

```diff
@@ -2722,9 +2722,25 @@ void MCInterfaceExecPopupWidget(MCExecContext &ctxt, MCNameRef p_kind, MCPoint *
 	
 	MCPoint t_at;
 	if (p_at != nil)
-		t_at = *p_at;
+    {
+        if (!MCtargetptr)
```

Comment:
@runrevmark I guess there's a chance of someone doing this:
- not handling a message in the target but somewhere further down
- deleting the target
- popping up a widget

Should we permit that?

Reply:
I think the current behavior of throwing an error is fine if there is no target - after all the script specifies a relative location which is now relative to the target, so if there is no target object anymore, then that doesn['t make any sense.

Reply:
ok